### DISCUSSION TITLE
I'm unable to see the DNS on the Domain page

### DISCUSSION THREAD

**Discussion by donah123** [vercel/community#2652] [Help] [answered]
### Summary

I'm unable to see DNS Record so as to set it. Can you help?
![image](https://github.com/vercel/community/assets/10235363/922d2045-f662-4684-a8d6-677d3271a1b7)

### Steps to Reproduce

Created a project

**Comment by smaeda-ks** [comment]
@donah123 What's the domain name? If the domain is already set up, you can check the DNS records using dig command:

```
$ dig +noall +ans example.com A
or 
$ dig +noall +ans sub.example.com CNAME
```

For Apex domain, use `A` record, and the value is `76.76.21.21 `. For other subdomains, use `CNAME` record, and the value is `cname.vercel-dns.com`.

**Comment by donah123** [comment, marked as answer]
The issue here is that I have the application on one Vercel account but
want to move it to another Vercel account. I'm getting an error message
about the domain. Another Vercel account is using this domain.Once the
verification is completed and the domain is successfully configured, the
TXT record can be removed.



On Tue, May 30, 2023 at 8:34 AM Shohei Maeda ***@***.***>
wrote:

> @donah123 <https://github.com/donah123> What's the domain name? If the
> domain is already set up, you can check the DNS records using dig command:
>
> $ dig +noall +ans example.com A
> or
> $ dig +noall +ans sub.example.com CNAME
>
> For Apex domain, use A record, and the value is 76.76.21.21 . For other
> subdomains, use CNAME record, and the value is cname.vercel-dns.com.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/2652#discussioncomment-6037748>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOC3Y46ZPFAATGZMTEVZELXIXSNDANCNFSM6AAAAAAYT3RQ6Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Olorunfemi Opeyemi.*

**Reply by amyegan** [reply]
If you're moving projects between a Hobby account and a Pro team, you can simply [transfer the project and domain](https://vercel.com/docs/concepts/projects/overview#transferring-a-project) rather than recreating it as a brand new project. If you're trying to move the project to a different Hobby account, you can add the TXT record with your DNS provider to verify that you own the domain. This record helps to keep your domain safe in the event that someone else was trying to hijack it for use on their own site.

**Comment by donah123** [comment]
Ok, thanks for the clarification.

On Tue, May 30, 2023 at 10:12 AM Amy ***@***.***> wrote:

> If you're moving projects between a Hobby account and a Pro team, you can
> simply transfer the project and domain
> <https://vercel.com/docs/concepts/projects/overview#transferring-a-project>
> rather than recreating it as a brand new project. If you're trying to move
> the project to a different Hobby account, you can add the TXT record with
> your DNS provider to verify that you own the domain. This record helps to
> keep your domain safe in the event that someone else was trying to hijack
> it for use on their own site.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/2652#discussioncomment-6038718>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOC3YZ6OVRPOEAMFQASJRLXIX55XANCNFSM6AAAAAAYT3RQ6Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Olorunfemi Opeyemi.*
